src/control.lua

```diff
@@ -911,30 +965,60 @@ function UpdateInvCombinators()
 		table.insert(invframe,{count=instance_id,index=#invframe+1,signal={name="signal-localid",type="virtual"}})
 	end
 
-	local items = game.item_prototypes
-	local fluids = game.fluid_prototypes
-	local virtuals = game.virtual_signal_prototypes
+	local items = lib_compat.version_ge("2.0.0") and prototypes.item or game.item_prototypes
+	local fluids = lib_compat.version_ge("2.0.0") and prototypes.fluid or game.fluid_prototypes
+	local virtuals = lib_compat.version_ge("2.0.0") and prototypes.virtual_signal or game.virtual_signal_prototypes
```

Comment:
If wanted, you can use `lib_compat.prototypes` here. e.g. `lib_compat.prototypes.item`